magnum_capi_helm/driver.py

```diff
@@ -691,7 +691,7 @@ def _storageclass_definitions(self, context, cluster):
 
         for volume_type in volume_types:
             storage_class = {
-                "name": volume_type,
+                "name": driver_utils.sanitized_name(volume_type),
```

Comment:
Isn't this your other unrelated change?

Reply:
yeah sorry, was layered on top as I was testing on tav1. fixed now